cmd/statreceiver/Dockerfile

```diff
@@ -0,0 +1,24 @@
+#Statreceiver Dockerfile
```

Comment:
This should look more like `cmd/versioncontrol/Dockerfile` We already handle building the binary in the Makefile, you just need to package it as a container image.